vanilla-framework/vanilla-framework
# Make icon mixins encode "#" by default

## Done

- Included `vf-url-friendly-color` function in `vf-icon` mixins, so that `#`s become `%23` (so Firefox can render them properly)
- Changed `vf-url-friendly-color` function to return `$color` on fail, for backwards compatibility
- Removed `vf-url-friendly-color` from accordion pattern

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-light/ **in Firefox**
- Check that the icons show up as normal
- Confirm that Percy shows no changes

## Details

Fixes #1856
